cranelift/fuzzgen/src/function_generator.rs

```diff
@@ -1863,7 +1863,7 @@ where
             .libcalls
             .iter()
             .map(|libcall| {
-                let signature = libcall.signature(lib_callconv);
+                let signature = libcall.signature(lib_callconv, I64);
```

Comment:
I think we have the target architecture available on `self` here and could compute the pointer size from it. I'm not too worried about it right now so you could leave a FIXME comment like you did in the interpreter if you prefer.